airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/debezium/CdcSourceTest.java

```diff
@@ -141,6 +142,15 @@ protected ConfiguredAirbyteCatalog getConfiguredCatalog() {
 
   protected abstract void assertExpectedStateMessages(final List<AirbyteStateMessage> stateMessages);
 
+  // TODO: this assertion should be added into test cases in this class, we will need to implement
+  // corresponding iterator for other connectors before
+  // doing so.
+  protected void assertExpectedStateMessageCountMatches(final List<AirbyteStateMessage> stateMessages, long totalCount) {
```

Comment:
Can we enable it for the relevant paths (Postgres/MySQL CDC) and disable it for others? That way we have coverage

Reply:
good call!